Rakefile

```diff
@@ -17,13 +17,12 @@ def create_database(uri)
   system command.join(' ')
 end
 
-def with_retries(count=5)
-  begin
-    yield if block_given?
-  rescue
-    raise if (count -= 1) == 0
-    sleep 2.25
-  end
+def with_retries(count = 5)
+  yield if block_given?
+rescue
+  raise if (count -= 1) == 0
+  sleep 2.25
+  retry
 end
```

Comment:
I'd never seen this syntax before, so I had to [look it up](https://github.com/ruby/ruby/blob/trunk/doc/syntax/exceptions.rdoc) to understand the behavior. Two things I learned:

1.
>If you are inside a method you do not need to use begin or end unless you wish to limit the scope of rescued exceptions:
```
def my_method
  # ...
rescue
  # ...
end
```

2.
> You may retry rescued exceptions:
```
begin
  # ...
rescue
  # do something that may change the result of the begin block
  retry
end
```
Execution will resume at the start of the begin block, so be careful not to create an infinite loop.